docs/source/interledger.rst

```diff
@@ -0,0 +1,57 @@
+Interledger Lab
+===============
+
+.. note:: **Work in progress**. 
+
+
+Build the services:
+
+.. code-block:: bash
+
+    $ docker-compose -f ledgers.yml build
+
+Run RethinkDB in the background:
+
+.. code-block:: bash
+
+    $ docker-compose -f ledgers.yml up -d rdb
+
+
+Configure each ledger:
+
+.. code-block:: bash
+
+    $ docker-compose -f ledgers.yml run bdb-0 bigchaindb -y configure
```

Comment:
Fixed with the new make file; it'll generate this file if it doesn't exist so docker doesn't accidentally make a directory instead.